### PR TITLE
Willys/384144-approvals

### PR DESCRIPTION
**Pull request by wpschaub**
Tips and tricks post to talk about the approval gates and artifact filters, as DevSecOps ,mentioned that we are not consistent across the board. This post will summrise our streamlined approval process for EDO Ce and the community.

### CODE REVIEW

**Comment by wpschaub** [comment]
@wsbctechnicalblog we need you and/or @rbeuthin, AND @604kev to review this PR before you pull the trigger.

Kevin, are you OK with the tone of the content talking about security engineers and does this summarize the pipeline hiccups we discussed?

**Review by wsbctechnicalblog — DISMISSED** [review]

File: content/pipelines-streamlined-approvals.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+>
+
+We take this quote to heart and have been mulling over our streamlines approval process. When we map a pipeline to a value stream it becomes evident that all of the gates and the security review contribute to our **Lead time for change** - WASTE.
+
+Where we can, we ...
+
+- Reduce the number and the sequence of approvals.
+- Automate validations and approvals.
+
+Leaving us with security review, which is based on automated vulnerability and quality scans and a few pairs of security engineer eyeballs. My colleague Kevin and I have been chatting about the [NEXUS](https://nexus-card.ca/)-type boarder control process, which may allow us to expedite parts of the security reviews based on **trust**. Watch this space for updates.
```
Comment (littleswancode) [inline content/pipelines-streamlined-approvals.md]: Typo @wpschaub :) "boarder" -> "border"

**Review by littleswancode — CHANGES_REQUESTED** [review]

**Review by littleswancode — DISMISSED** [review]
Typo resolved, looks good!

**Comment by wpschaub** [comment]
@wsbctechnicalblog once you are happy, you can pull the trigger on this streamlined approvals post.

**Review by wsbctechnicalblog — COMMENTED** [review]
The date of this post is 2021-03-13. Is that correct?

**Comment by wpschaub** [comment]
@wsbctechnicalblog date has been fixed to 11/2. Thank you for the catch!

**Review by wsbctechnicalblog — APPROVED** [review]

**Pull request merged**